### PR TITLE
SPL-140: pa11y warning - The heading structure is not logically nested.

### PR DESCRIPTION
**Pull request by nigeldunne**

### CODE REVIEW

File: app/views/components/calendar.njk [hunk 1]
```diff
@@ -1,103 +1,103 @@
...
+  {% set minWeek = data | earliestPrimaryLeaveWeek %}
+  {% set maxWeek = 52 %}
+  {% set zeroWeekDate = (data | startDay | startOfWeek) %}
+  {% set isBirth = (data | isBirth) %}
+  {% set isAdoption = (data | isAdoption) %}
+  {% set weekZeroName= (data | birthOrPlacement) %}
+  <div class="govuk-form-group">
+    {% call govukFieldset({
+      legend: {
+        html: "<h2 class=\"govuk-heading-m govuk-!-margin-0\">" + (data | parentName(parent) | capitalize) + "</h2>",
```
Comment (velimd) [inline app/views/components/calendar.njk:13]: class name should be heading-l as its h2 now

File: app/views/components/calendar.njk [hunk 2]
```diff
@@ -1,103 +1,103 @@
...
+          {% for i in range(minWeek, maxWeek + 1) %}
+            {% set compulsoryLeave = (parent === 'primary' and isBirth and (i === 0 or i === 1)) %}
+            {% set outOfRange = (parent === 'secondary' and i < 0) %}
+            {% set leaveId = parent + '-leave_' + i %}
+            {% set leaveName = parent + '[leave]' %}
+            {% set payId = parent + '-pay_' + i %}
+            {% set payName = parent + '[pay]' %}
+            {% set date = (zeroWeekDate | offsetWeeks(i)) %}
+            {% if (i === minWeek) or ((date | formatDate("D") | int) <= 7) %}
+              <h3 class="govuk-heading-s govuk-!-padding-1 background-grey">
```
Comment (velimd) [inline app/views/components/calendar.njk:28]: heading-m

File: app/views/components/calendar.njk (hunk 1 above)
Comment (nigeldunne) [reply app/views/components/calendar.njk:13]: I don't want to change the appearance, just fix the semantics of the html

File: app/views/components/calendar.njk (hunk 2 above)
Comment (nigeldunne) [reply app/views/components/calendar.njk:28]: I don't want to change the appearance, just fix the semantics of the html

**Pull request merged**